src/sign/non_interactive_sign/participant.rs

```diff
@@ -626,6 +629,22 @@ mod test {
                 distributed_sig.as_ref()
             )
             .is_ok());
+
```

Comment:
This code ensures we can successfully generate the recovery ID and derive the public key from that recovery ID on this unit test.